tests/monitoring/monitoring.go

```diff
@@ -323,23 +395,23 @@ var _ = Describe("[Serial][sig-monitoring]Prometheus Alerts", func() {
 	Context("Up metrics", func() {
 		BeforeEach(func() {
 			scales = make(map[string]*autoscalingv1.Scale, 3)
-			backupScale(virtOperator.deploymentDame)
-			backupScale(virtController.deploymentDame)
-			backupScale(virtApi.deploymentDame)
+			backupScale(virtOperator.deploymentName)
+			backupScale(virtController.deploymentName)
+			backupScale(virtApi.deploymentName)
 		})
 
 		AfterEach(func() {
-			revertScale(virtApi.deploymentDame)
-			revertScale(virtController.deploymentDame)
-			revertScale(virtOperator.deploymentDame)
+			revertScale(virtApi.deploymentName)
+			revertScale(virtController.deploymentName)
+			revertScale(virtOperator.deploymentName)
 
 			time.Sleep(10 * time.Second)
 			waitUntilAlertDoesNotExist("VirtOperatorDown")
 			waitUntilAlertDoesNotExist("NoReadyVirtOperator")
```

Comment:
Since I follow your improvement logic, why don't you add
```
			waitUntilAlertDoesNotExist(virtController.downAlert)
			waitUntilAlertDoesNotExist(virtApi.downAlert)
```
here as well?

Reply:
added